globe_search.py

```diff
@@ -0,0 +1,40 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.options import Options
+import time
+
+def searcher_name_games(game: str) -> list:
+    game_html_format = game.replace(' ', '+')
+    search_game = game_html_format.lower()
```

Comment:
You can do that without introducing any new variables
game = game.replace().lower()...

Reply:
solved